test/components/MusicPlaylist/MusicPlaylist1Test.java

```diff
@@ -12,7 +12,7 @@ public class MusicPlaylist1Test {
      */
     @Test
     public final void testShuffleZero() {
-        MusicPlaylist n = new MusicPlaylist();
+        MusicPlaylist n = new MusicPlaylist1();
         MusicPlaylist nCopy = n;
```

Comment:
This test and others like it are bugged. They create an alias, so the `.equals()` will always come back true.